Initial commit

SteveMattison opened the pull request:
* Added basic functionality of a lambda to create a iiif manifest on-the-fly.
* Legal resources include: 'manifest', 'canvas', 'annotation_page', and 'annotation'
* Added appropriate tests

SteveMattison committed:
Initial Commit

Added basic functionality of lambda.
Legal resources include: 'manifest', 'canvas', 'annotation_page', and 'annotation'

SteveMattison committed:
Refactored to simplify

* Refactored to eliminate passing objects within objects
* Attempted to make modules easier to test atomically

SteveMattison committed:
Added unit tests

SteveMattison committed:
Delete setup.py

SteveMattison committed:
Update dev-requirements.txt

rdoughty commented:
do we need this manifest_lambda/dependencies folder to be checked in? usually that’s done via the deployment process and those dependencies are not in the codebase

SteveMattison committed:
Update .gitignore

Ignore dependencies folder

SteveMattison committed:
Removed dependencies files

Removed files from /dependencies that should never have been committed in the first place.
